invenio_rdm_pure/version.py

```diff
@@ -11,4 +11,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = "1.0.0.dev20200000"
+__version__ = "0.1.0"
```

Comment:
why change the version - but not a big deal.

Reply:
I just tried to keep it consistent with https://github.com/inveniosoftware/cookiecutter-invenio-module. This was changed in
e3ca55f18a3e35ed4d3f5875d9647d408c45295b